### ISSUE TITLE
👨‍👩‍👧‍👦 Per volunteer page

### ISSUE THREAD

**Issue by jofftiquez** [OSSPhilippines/ossph.org#41]
Create a volunteer profile page for each volunteer

**Comment by danielfpadilla** [comment]
Hello, what would be the url path to each volunteer profile page?

**Comment by jofftiquez** [comment]
Yes sir, but currently this is assigned to me. I'm working on a new design rn. 😊
